test/blockgaslimit.js

```diff
@@ -0,0 +1,73 @@
+const Web3 = require("web3");
+const assert = require("assert");
+const Ganache = require("../index");
+const path = require("path");
+const compileAndDeploy = require("./helpers/contracts").compileAndDeploy;
+
+const mnemonic = "candy maple cake sugar pudding cream honey rich smooth crumble sweet treat";
+
+function setUp(options = { mnemonic }, contractName = "BlockGasLimit") {
+  let context = {
```

Comment:
We can declare this as const.  Same with lines 41-43.

Reply:
Update is on the way.